### DISCUSSION TITLE
Codeium Chat UI Improvement

### DISCUSSION THREAD

**Discussion by EHMilon** [VSCodium/vscodium#1702] [Ideas]
Sidebar space is already less. Now the space between sidebar and chat, I think it taking extra space.
And in the codeium chat bottom, lots of text and icons make it awkward. I think **tips** icon should be added top of the search box beside conversion history and under the search box instructions should be removed.
If you look carefully at the image, you will discover that placeholder text is getting out from search box.

<img width="232" alt="image" src="https://github.com/VSCodium/vscodium/assets/107425940/91ba0db0-9668-4ab6-864c-f9f78b1321ef">

**Comment by daiyam** [comment]
This is an issue about the Codeium extension and not VSCodium editor.

**Comment by EHMilon** [comment]
> This is an issue about the Codeium extension and not VSCodium editor.

Is i am in the wrong place?
